rootconf/default/bin/onie-support

```diff
@@ -58,6 +58,11 @@ export > $save_dir/runtime-export-env.txt
 set > $save_dir/runtime-set-env.txt
 ps w > $save_dir/runtime-process.txt
 dmesg > $save_dir/dmesg.txt
+onie-syseeprom > $save_dir/onie-syseeprom.txt
+onie-sysinfo > $save_dir/onie-sysinfo.txt
```

Comment:
It would be better to use the `-a` option to `onie-sysinfo` as that dumps all the info.